Softeq.XToolkit.WhiteLabel.Droid/ViewComponents/BottomNavigationComponentBase.cs

```diff
@@ -105,16 +105,14 @@ private void InflateMenu(Context context)
             }
         }
 
-        private void BottomNavigationViewNavigationItemSelected(
-            object sender,
-            BottomNavigationView.NavigationItemSelectedEventArgs e)
+        private void BottomNavigationView_ItemSelected(object sender, Google.Android.Material.Navigation.NavigationBarView.ItemSelectedEventArgs e)
```

Comment:
Can't we simplify type name with using? Also, I'm not sure if we should use ubderscore in method name acording to our guidlines

Reply:
fixed